build.gradle

```diff
@@ -16,16 +16,9 @@ allprojects {
 
     apply from: "$rootDir/gradle/quality.gradle"
 
-    // ---
-    // To create a release release:
-    // 1. Merge a PR with "-SNAPSHOT" suffix removed
-    // 2. Merge a second PR which bumps the version with "-SNAPSHOT" suffix added back
-    // For example, given a version of "1.2.3-SNAPSHOT", merge "1.2.3" then "1.2.4-SNAPSHOT".
-    // ---
-    // In general, users of this SDK should NOT depend on "-SNAPSHOT" builds, as they may break
-    // at ANY TIME as changes are merged into master. They represent a fully unstable API.
-    // ---
```

Comment:
Good point, updated this section to retain background info and step-by-step under the new system